content/en/platform/corda/5.1/developing-applications/tooling/_index.md

```diff
@@ -19,13 +19,15 @@ table th:nth-of-type(2) {
 }
 
 </style>
+
 # Application Developer Tooling
-{{< tooltip >}}CorDapp{{< /tooltip >}} Developers require the following:
 
+{{< tooltip >}}CorDapp{{< /tooltip >}} Developers require the following:
 
 | Tool                                            | Description                                                                                                                                                                                                                                                                                                |
 | ----------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| CorDapp Standard Development Environment (CSDE) | The [CSDE]({{< relref "../../../../../tools-corda5/csde/_index.md" >}}) guides Developers who are new to Corda 5 from setting up their development environment through to writing, compiling, and running their first basic CorDapp.                                                                                     |
-| {{< tooltip >}}Corda CLI{{< /tooltip >}}                                       | The CSDE uses the Corda CLI in the background. CorDapp Developers also require the Corda CLI to build {{< tooltip >}}CPB{{< /tooltip >}} files. For information about installation, see [Installing the Corda CLI]({{< relref "./installing-corda-cli.md" >}}).                                 |
+| CorDapp Standard Development Environment (CSDE) | The [CSDE]({{< relref "../../../../../tools-corda5/csde/_index.md" >}}) guides Developers who are new to Corda 5 from setting up their development environment through to writing, compiling, and running their first basic CorDapp.                                                                       |
```

Comment:
```suggestion
| CorDapp Standard Development Environment (CSDE) | The [CSDE]({{< relref "../../../../../tools-corda5/csde/_index.md" >}}) guides developers who are new to Corda 5 through the stages from initially setting up their development environment through to writing, compiling, and running their first basic CorDapp.                                                                       |
```

Added this text as when I read it initially I was a bit confused by the use of "from"....